e2e_tests/test_routes.py

```diff
@@ -0,0 +1,42 @@
+from playwright.sync_api import Page
+
+
+class TestRoutes:
+    def test_cookies_get(self, standard_user_page: Page):
```

Comment:
I noticed we have this e2e test and very similar integration test (using client.get). They both assert content in the html but slightly different assertions. I think we could do with just one of these, I dont think having both adds anything as long as we move the assertions from one to the other.

Reply:
The idea is to create e2e tests for all static content pages to have full testing coverage, this is just the start as it triggered coverage failure when I created a PR and it come as not covered